lib/flight.js

```diff
@@ -124,3 +124,13 @@ Flight.prototype.rightFlip = function() {
 Flight.prototype.wave = function() {
   return this.connection.animate("wave", 750);
 };
+
+/**
+ * Tells the drone to perform horizontal trimming.
+ *
+ * @return {null}
+ * @publish
+ */
+Flight.prototype.ftrim = function() {
```

Comment:
The only reason they should be, is if they have a different API signature, or are a higher-level abstraction. Anything that is a straight proxy does not need to be in there.